tcms/testplans/templates/testplans/get.html

```diff
@@ -15,6 +15,8 @@ <h1 class="col-md-12" style="margin-top: 0">
         <span id="test_plan_pk"
               data-testplan-pk="{{ object.pk }}"
               data-perm-remove-tag="{{ perms.testplans.delete_testplantag }}"
+              data-perm-change-testcase="{{ perms.testcases.change_testcase }}"
+              data-perm-delete-testcase="{{ perms.testcases.delete_testcaseplan }}"
```

Comment:
`perms.testcases.delete_testcaseplan` - this is the permission to remove a TC from TP, not to delete the TC from the DB entirely.